src/main/java/com/yelp/nrtsearch/server/luceneserver/SearchHandler.java

```diff
@@ -323,6 +323,48 @@ public SearchResponse handle(IndexState indexState, SearchRequest searchRequest)
     return searchResponse;
   }
 
+  public void handleStrippedWarmingQuery(IndexState indexState, SearchRequest searchRequest) {
```

Comment:
It seems like this function is unused, and only `WarmingUtils.handleStrippedWarmingQuery` is called